eidas-saml-parser/src/main/java/uk/gov/ida/notification/eidassaml/EidasSamlParserConfiguration.java

```diff
@@ -36,4 +40,9 @@ public TrustStoreBackedMetadataConfiguration getConnectorMetadataConfiguration()
     public URI getProxyNodeAuthnRequestUrl() {
         return proxyNodeAuthnRequestUrl;
     }
+
+    @Override
+    public boolean isPrometheusEnabled() {
+        return prometheusEnabled;
```

Comment:
```suggestion
        return true;
```
related to above comment